### DISCUSSION TITLE
Extend Pipeline using Config File

### DISCUSSION THREAD

**Discussion by maximilianreimer** [explosion/spaCy#10035] [Help: Coding & Implementations] [answered]
I would like to extend an existing trained pipeline and be able to use the command-line interface to run things like an evaluation on it. 

Is there a way of doing this? 

So the code looks currently like this:

```python 
nlp = spacy.load("en_core_web_trf")
nlp.add_pipe("sentencizer")
```

I know that you can evaluate the  `en_core_web_trf` pipeline using `python -m spacy evaluate  en_core_web_trf data/path`. What is the best way to run this for the extended pipeline?

I was hoping that I could create a config file that looks somewhat like this: 

```
#mypipeline.cfg
[nlp]
lang = "en"
pipeline = ["en_core_web_trf","sentencizer"]
```

and run `python -m spacy evaluate  mypipeline data/path`. 

Thanks

**Comment by polm** [comment, marked as answer]
If you create your pipeline in code like that and save it to disk you can specify it from the command line just like `en_core_web_trf`. Just pass the path to the model you saved instead of `en_core_web_trf`.
